triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -168,18 +168,24 @@ object Server {
           }
         )
       },
-      // List triggers currently running for the given party
       get {
-        path("list") {
-          entity(as[ListParams]) { params =>
-            {
-              val triggerList =
-                triggersByParty.getOrElse(params.party, Set()).map(_.toString).toList
-              val result = JsObject(("triggerIds", triggerList.toJson))
-              complete(successResponse(result))
+        // Convenience endpoint for tests
```

Comment:
How about giving this a bit more sensible name. E.g. `/healthz` is a pretty common name for health check endpoints and lots of tooling defaults to it.

Reply:
I didn't know that :)

Reply:
One example https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/ but if you search for it you can find lots of other examples from prometheus and whatnot.

Reply:
Quick Google suggests `health`. That's what you meant right?

Reply:
I did mean `healthz` see the link but `health` works as well. Not sure what is more common in general.